.changeset/kind-apples-itch.md

```diff
@@ -0,0 +1,21 @@
+---
+'eslint-plugin-prettier': minor
+---
+
+feat: support ESLint flat config system
+
+`eslint-plugin-prettier` can be used in the flat config format like this with CJS:
+
+```js
+const prettier = require('eslint-plugin-prettier/flat');
+
+module.exports = [prettier.configs.recommended];
+```
+
+Or with ESM:
+
+```js
+import prettier from 'eslint-plugin-prettier/flat';
+
+export default [prettier.configs.recommended];
```

Comment:
It's still strange to me that only `prettier.configs.recommended` is used but the whole `prettier` plugin object is exported, however the plugin inside is actually the legacy one.